java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -1530,23 +1295,15 @@ public TypeCastExpressionTreeImpl newBasicTypeCastExpression(PrimitiveTypeTreeIm
 
   public TypeCastExpressionTreeImpl newClassCastExpression(TypeTree type, Optional<List<Tuple<InternalSyntaxToken, Tree>>> classTypes, InternalSyntaxToken closeParenToken, ExpressionTree expression) {
     ImmutableList.Builder<Tree> boundsBuilder = ImmutableList.builder();
-    List<AstNode> children = Lists.newArrayList();
-    children.add((AstNode) type);
     if (classTypes.isPresent()) {
       for (Tuple<InternalSyntaxToken, Tree> tuple : classTypes.get()) {
         // TODO SONARJAVA-547 andOperator should be present in the tree
         InternalSyntaxToken andOperator = tuple.first();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "andOperator" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)